extensions/currency-exchange/src/index.tsx

```diff
@@ -384,14 +387,13 @@ function enrichExchangeData(
   return currencyData as CurrencyResult;
 }
 
-function getFlagEmoji(countryCode: string): string {
-  const codePoints = countryCode
+function getFlagEmoji(countryAndRegionCode: string): string {
+  const codePoints = countryAndRegionCode
     .toUpperCase()
     .split("")
     .map((char) => 127397 + char.charCodeAt(0));
-  let v = countryCode.startsWith("X") ? "🇺🇳" : String.fromCodePoint(...codePoints) || "🇺🇳";
-  v = countryCode === "TW" ? "🇨🇳" : v;
-  v = countryCode === "AN" ? "🇺🇳" : v;
+  let v = countryAndRegionCode.startsWith("X") ? "🇺🇳" : String.fromCodePoint(...codePoints) || "🇺🇳";
+  v = countryAndRegionCode === "AN" ? "🇺🇳" : v;
```

Comment:
I am pretty sure penguins will not come here and talk about their flag, but what’s wrong with using the 🇦🇶 flag? In fact, all of this extra processing is pointless.